src/postgres/src/test/regress/yb_pg_join_schedule

```diff
@@ -0,0 +1,20 @@
+# src/test/regress/yb_pg_join_schedule
+#
+####################################################################################################
+# This includes tests for Join Clause
+####################################################################################################
+# Dependencies for `yb_pg_join`
+test: yb_pg_int4
+test: yb_int2
+test: yb_pg_int8
+test: yb_float8
+test: yb_text
```

Comment:
I don't understand why `yb_foo` tests are dependencies for `yb_pg_join`.  Can you explain?

Reply:
Even `yb_int2 `and` yb_text` turned out to be ported tests. I've renamed them too accordingly.